cmd/aws-janitor/main.go

```diff
@@ -143,3 +183,24 @@ func markAndSweep(opts resources.Options, region string) error {
 
 	return nil
 }
+
+func pushMetricBeforeExit(pusher *push.Pusher, startTime time.Time, jobType bool, exitCode int) {
+	// Set the status of the job
+	status := "failed"
+	if exitCode == 0 {
+		status = "success"
+	}
+
+	// Set the type of the job to report the metric
```

Comment:
I think a better label here might be `clean_all` which is then either set to `true` or `false` depending on the flag setting.

Reply:
(also, since it's a global variable, you may not need to pass along the setting of `cleanAll`)

Reply:
makes sense, thanks for the feedback